controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -3359,7 +3359,8 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
             return false;
         }
 
-        auto radio_mac = notification->radio_mac();
+        //get the mac from hostap_mac
+        auto radio_mac = network_utils::mac_from_string(hostap_mac);
```

Comment:
Since this change would effect more then just the DCS feature,
I'm going to create a new issue for it (assigned to me), and I'll add a Hotfix to solve this once I'm finished with my current issues